connectors/src/api/webhooks/webhook_slack.ts

```diff
@@ -41,6 +43,53 @@ type SlackWebhookResBody =
   | null
   | APIErrorWithStatusCode;
 
+async function handleChatBot(req: Request, res: Response) {
+  const slackMessage = req.body.event.text;
```

Comment:
optional: you could do a bit more concise this way
const slackRequest = { message: , teamId: , ..., messageTs: req.body.event.ts}

if (Object.values(slackRequest).some(v => !v)) { // error "missing fileds"

const botRes = await botAnswer..(slackRequest)
logger.error({error: ..., slackRequest)